steps/aggregate.sh

```diff
@@ -98,4 +76,24 @@ done < "${repos}"
 "${LOCAL}/help/parallel.sh" "${jobs}"
 wait
 
+mkdir -p "${TARGET}/data/aggregation"
+f_jobs=${TARGET}/temp/jobs/aggregate-function-jobs.txt
+rm -rf "${f_jobs}"
+mkdir -p "$(dirname "${f_jobs}")"
+touch "${f_jobs}"
+
+for metric in ${metrics}; do
+    metric_file="${TARGET}/data/${metric}.csv"
+    if [[ -f "${metric_file}" ]]; then
+        output_folder="${TARGET}/data/aggregation"
+        for sh_script in "${LOCAL}/steps/aggregation-functions/"*.sh; do
+            if [[ -f "${sh_script}" ]]; then
+                printf "%s %s %s %s\n" "${sh_script@Q}" "${metric_file}" "${output_folder@Q}" "${metric@Q}" >> "${f_jobs}"
+            fi
+        done
+    fi
+done
+"${LOCAL}/help/parallel.sh" "${f_jobs}"
+wait
```

Comment:
@dzhovi why did you move this block to the bottom of the file? I can't understand the reason for this change

Reply:
@yegor256 Aggregation requires csv files for each metric, like LCOM5.csv, which store values for each class. I did not face that problem when i was testing it previously, as these files were generated from previous runs. But doing clear run will fail with a error, so i put this block in the end to have a guarantees over metric files existence